### DISCUSSION TITLE
I reported an error when calling wallet connect2.0 using ethereum provider and model

### DISCUSSION THREAD

**Discussion by jiangeWu** [WalletConnect/walletconnect-monorepo#2595] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/25365253/e74b657e-6167-45af-91a6-d7c1858fde95)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/25365253/7c7d34c3-096a-4804-b647-d4565479b7d0)
I don't know why the error was reported

**Comment by finessevanes** [comment]
Hey, it looks like this might be a duplicate report on this [issue](https://github.com/orgs/WalletConnect/discussions/2594).

Could you walk me through the step to reproduce? As well as the `ethereum-provider` version you're using?
